src/govuk/components/modal-dialogue/_modal-dialogue.scss

```
@@ -0,0 +1,161 @@
+@import "../../settings/all";
+@import "../../tools/all";
+@import "../../helpers/all";
+
+@include govuk-exports("govuk/component/modal-dialogue") {
+  $govuk-dialogue-width: 640px;
+
+  .govuk-modal-dialogue,
+  .govuk-modal-dialogue__backdrop {
+    position: fixed;
+    z-index: 0;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  // Hide dialogue when closed
+  .govuk-modal-dialogue {
+    display: none;
+  }
+
+  // Show dialogue when opened
+  .govuk-modal-dialogue--open {
+    display: block;
+  }
+
+  // Wrapper to handle overflow scrolling
+  .govuk-modal-dialogue__wrapper {
+    box-sizing: border-box;
+    display: flex;
+    height: 100%;
+    @include govuk-responsive-padding(7, "top");
+    @include govuk-responsive-padding(7, "bottom");
+    overflow-y: auto;
+    align-items: flex-start; // sass-lint:disable no-duplicate-properties
+    align-items: safe center;
+  }
+
+  // HTML5 dialogue component
+  .govuk-modal-dialogue__box {
+    box-sizing: border-box;
+    display: block;
+    position: relative;
+    z-index: 1;
```

We might want to increase this, just in the (non-ideal) case that the page has other competing elements. I notice that the [GOV.UK Publishing equivalent sets this to 1000](https://github.com/alphagov/govuk_publishing_components/blob/34315777fabc6ac53348e636c7d3006b62228fcd/app/assets/stylesheets/govuk_publishing_components/components/_modal-dialogue.scss#L3).

---

I've made this 1001 so it is above the backdrop which also needs the z-index increasing.
```suggestion
    z-index: 1001;
```